pkg/apiserver/apiserver.go

```diff
@@ -268,8 +268,6 @@ func (g *APIGroupVersion) InstallREST(container *restful.Container, root string,
 
 	// TODO: port the rest of these. Sadly, if we don't, we'll have inconsistent
 	// API behavior, as well as lack of documentation
-	mux := container.ServeMux
```

Comment:
The change here is that instead of directly using ServeMux, we want to use MuxHelper. MuxHelper will record that we are handling this path and then forward the call to ServeMux. This is required because net/http does not expose the paths handled by ServeMux.
And as Brian has pointed out, we want to soon convert all mux.Handle calls in this method to use restful.WebService.
Restful exposes all the paths handled by it. But we will still need MuxHelper for mux.Handle calls outside this method.